lib/formats/map.js

```diff
@@ -23,6 +25,32 @@ export const mapEs = {
   },
 };
 
+
+/**
+ * JavaScript token map
+ * @type {import('style-dictionary').Format}
+ * @example ```js
+ * import { tokensMeta } from '@rhds/tokens/meta.js';
+ *
+ * const fontfamilyCssString = tokensMeta.get('--rh-font-family-body-text');
+ * // e.g. '{
+ * value: 'RedHatText, "Red Hat Text", sans-serif',
+ * description: 'The font family used for body text',
+ * extensions: ''
+ * }';
+ * ```
```

Comment:
```suggestion
 * @example Importing token objects in JavsScript modules
 *          ```js
 *          import { tokensMeta } from '@rhds/tokens/meta.js';
 *
 *          const family = tokensMeta.get('--rh-font-family-body-text');
 *          console.log(family.$description);
 *          // 'The font family used for body text'
 * ```
```

indent trick to get jsdoc to render properly in IDE hover. otherwise, leading whitespace for each line would be trimmed